src/components/main/Summary.tsx

```diff
@@ -0,0 +1,55 @@
+import Card from "../UI/Card"
+import axios from 'axios'
+import { useQuery } from '@tanstack/react-query'
+import { motion } from 'framer-motion'
+
+const Summary = () => {
+    const fetchDescription = async () => {
```

Comment:
you could extract this into a hook, should not be here

Reply:
Ok done. However I'm not sure if the error handling here is appropriate (or, optimized)